t/05-scheduler-dependencies.t

```diff
@@ -141,6 +143,33 @@ subtest 'assign multiple jobs to worker' => sub {
     ok($job_token, 'job token present');
 };
 
+# prevent writing to a log file to enable use of combined_like in the following tests
+my $usual_log = $t->app->log;
+$t->app->log(Mojo::Log->new(level => 'debug'));
```

Comment:
Maybe this should just be enabled at the start of the test? 🤔

Reply:
No, it shouldn't. I also intentionally disable it immediately after the tests using it again. That's because other logs messages would otherwise interfere with the test output (unless explicitly handled).

Reply:
can you do that within the subtest as well as the restoring?